src/app/plugins/ta.plugin.ts

```diff
@@ -148,7 +148,16 @@ export default class TaPlugin extends Plugin {
     }
 
     const mentions = TAs.map((m) => m.user.toString()).join(' ');
-    message.channel.send(`${mentions}\n${message.author} asks: \`\`\`${question}\`\`\``);
+    await message.channel.send(`${mentions}`);
+
+    const embed: MessageEmbed = new MessageEmbed()
+    .setColor('#0099ff')
+    .setAuthor(message.author.tag, message.author.displayAvatarURL())
+    .setDescription(`${question}`)
+    .setTimestamp();
+
+    await message.channel.send({ embeds: [embed] });
```

Comment:
```suggestion
    const embed: MessageEmbed = new MessageEmbed()
    .setColor('#0099ff')
    .setAuthor(message.author.tag, message.author.displayAvatarURL())
    .setDescription(`${question}`)
    .setTimestamp();

    await message.channel.send({ content: `${mentions}`, embeds: [embed] });
```

Reply:
Thanks, I didn't know that was possible.